Fix accidental usages of RST's default role (#12272)
godotengine/godot-docs: tutorials/2d/introduction_to_2d.rst

  ``x: 1, y: 1``.
- **Use Pixel Snap**: Toggles the use of subpixels for snapping. If enabled, the position values
  will be integers, disabling will enable subpixel movement as decimal values. For the runtime
  property, consider checking :ui:`Project Settings > Rendering > 2D > Snapping` property for

>   Node2D nodes, and :ui:`Project Settings > GUI > General > Snap Controls to Pixels` for

Reviewer: ```suggestion
  property, consider checking :menu:`Project Settings > Rendering > 2D > Snapping` property for
  Node2D nodes, and :menu:`Project Settings > GUI > General > Snap Controls to Pixels` for
```

Author: Especially check this again, because I am a bit perplexed.

Middle-clicking and dragging the mouse will pan the view.
The scrollbars on the right or bottom of the viewport also move the view.
Alternatively, the :kbd:`G` or :kbd:`Space` keys can be used.

> If you enable :ui:`Editor Settings > Editors > Panning > Simple Panning`, you can activate

Reviewer: ```suggestion
If you enable :menu:`Editor Settings > Editors > Panning > Simple Panning`, you can activate
```

repositioned by dragging them back to their respective rulers, and they can be removed by
dragging them all the way back to the ruler.

> You can also enable snapping to the created guides using the :menu:`Smart Snap` menu.

Reviewer: This isn't how the menu role is supposed to be used as I understand it. It's explicitly for clicking through multiple layers. I think for stuff like this you'd just use ":ui:"

https://contributing.godotengine.org/en/latest/documentation/guidelines/docs_writing_guidelines.html#use-roles-for-editor-ui

I think I saw other instances in this PR like this, those should be changed as well.

Author: There was some ambiguity in our guidelines that I was genuinely not sure about until your comment. It's "*when* listing a series", not "to list a series", implying it's not always the case.

  can also toggle them with the provided option.
- **Show Helpers**: Toggles the temporary display of an outline of the node, with the previous
  transform properties (position, scaling, or rotation) if a transform operation has been
  initiated. For Control nodes, it also shows the sizing parameters. Useful to see the deltas.
- **Show Rulers**: Toggles the visibility of horizontal and vertical rulers. See
  :ref:`doc_introduction_to_2d_the_viewport` more on rulers.
- **Show Guides**: Toggles the visibility of created guides. See
  :ref:`doc_introduction_to_2d_the_viewport` for on how to create them.
- **Show Origin**: Toggles the display of the green and red origin lines drawn at ``x: 0, y: 0``.
- **Show Viewport**: Toggles the visibility of the game's default
  viewport, indicated by an indigo-colored rectangle. It is also the default window size on desktop

>   platforms, which can be changed by going to :ui:`Project Settings > Display > Window > Size` and

Reviewer: ```suggestion
  platforms, which can be changed by going to :menu:`Project Settings > Display > Window > Size` and
```
